### DISCUSSION TITLE
Est-ce qu'il est possible d'activer 2 attributs HTML sur le script tarteaucitron ?

### DISCUSSION THREAD

**Discussion by Jjodar** [AmauriC/tarteaucitron.js#1093] [Q&A]
Est-ce qu'il est possible d'activer 2 attributs HTML sur script tarteaucitron ? (c'est pour optimiser la vitesse de chargement)

Merci d'avance pour les réponses. 

Jonathan

**Comment by AmauriC** [comment]
Quels attributs? Un defer ou async? Ca peux poser des problèmes pendant le chargement de tarteaucitron.

**Reply by Jjodar** [reply]
J'ai utilisé les deux méthodes. Ainsi, je n'ai rencontré aucun problème de chargement du script et le panneau s'affiche correctement.
Je voudrais plutôt savoir s'il est recommandé de procéder ainsi ou de ne pas le faire.
Dans tous les cas, j'ai réussi à gagner un certain nombre de millisecondes lors du chargement du site.
Jonathan

**Reply by AmauriC** [reply]
Si tout fonctionne bien, pas de souci :)
Il n'y a que si vous utilisez le mode auto de la v pro où ça pose problème, tarteaucitron ne pourra pas tout détecter.
Si mode manuel, c'est tout bon.

**Reply by Jjodar** [reply]
Merci pour les réponses. C'est top.
